test/helpers/setup-browser-env.js

```diff
@@ -0,0 +1,2 @@
+import browserEnv from 'browser-env';
```

Comment:
this imports a dom environment, it is needed for ava.js. Basically it uses jsdom library under the hood.